index.html

```diff
@@ -51,17 +51,12 @@ <h1>Technologist and<br>Software Engineer<br>based in Bristol</h1>
 
         <section class="intro-section"> 
 
-            <div>
+            <div class="intro-section-message-div">
```

Comment:
it would be more semantic to drop the `div` from the class name - maybe `intro-section-message` and then you can target the `<p>` inside it using `intro-section-message p`

or alternatively you can call this `intro-section-message-parent` because that way you can change the tag to something else and not have to change the class name